homeassistant/components/sharkiq/config_flow.py

```diff
@@ -85,7 +85,8 @@ async def async_step_reauth(self, user_input: dict | None = None):
 
             if not errors:
                 entry = await self.async_set_unique_id(self.unique_id)
-                self.hass.config_entries.async_update_entry(entry, data=user_input)
+                if entry:
+                    self.hass.config_entries.async_update_entry(entry, data=user_input)
```

Comment:
If we can't update the existing entry we should show an error. It's not a success unless we update the existing entry with new credentials.